nautobot_ssot/integrations/device42/diffsync/models/base/ipam.py

```diff
@@ -45,10 +45,11 @@ class IPAddress(DiffSyncModel):
 
     _modelname = "ipaddr"
     _identifiers = ("address", "vrf")
-    _attributes = ("available", "label", "device", "interface", "primary", "tags", "custom_fields")
+    _attributes = ("subnet", "available", "label", "device", "interface", "primary", "tags", "custom_fields")
     _children = {}
 
     address: str
+    subnet: str
```

Comment:
Should this be `subnet_mask` or `prefix_length`? Or is it a link to the corresponding prefix?

Reply:
FK to parent prefix

Reply:
This is to hold the actual prefix for designating the parent prefix during IP Address creation.